src/sb_utils/include/RvizUtils.h

```diff
@@ -73,6 +77,97 @@ class RvizUtils {
                  int type = visualization_msgs::Marker::POINTS,
                  int id   = 0);
 
+    /**
+     *  Turn a point into a marker for rviz
+     *
+     *  @param point the point to be converted
+     *  @param color the color of the point
+     *  @param frame_id the frame id
+     *  @param ns the namespace
+     *
+     *  @return an rviz marker
+     */
+    visualization_msgs::Marker
+    createMarker(geometry_msgs::Point point,
+                 visualization_msgs::Marker::_color_type color,
+                 visualization_msgs::Marker::_scale_type scale,
+                 std::string frame_id,
+                 std::string ns,
+                 int type = visualization_msgs::Marker::POINTS,
+                 int id   = 0);
+
+    /**
+     *  Turn a polygon into a marker for rviz
+     *
+     *  @param polygon the polygon to be converted
+     *  @param color the color of the polygon
+     *  @param frame_id the frame id
+     *  @param ns the namespace
+     *
+     *  @return an rviz marker
+     */
+    template <typename T>
+    visualization_msgs::Marker
+    createPolygonMarker2D(T polygon,
+                          visualization_msgs::Marker::_color_type color,
+                          visualization_msgs::Marker::_scale_type scale,
+                          std::string frame_id,
+                          std::string ns,
+                          int type = visualization_msgs::Marker::LINE_STRIP,
+                          int id   = 1) {
```

Comment:
I don't know the proper conventions for template functions, so I am not sure how to make it look good.

Reply:
Why are we templating here?

Reply:
At the time I thought it'd be useful when working with any type of point. But thinking about it now we should always be using the snowbots specific points. I will change it back.